Snowflake.Data.Tests/Client/AllTypesClass.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+namespace Snowflake.Data.Tests.Client
+{
+    public class AllTypesClass
```

Comment:
what is the purpose?

Reply:
I renamed the class to AllUnstructuredTypesClass. It is in scope of tests only and used in tests to verify how Snowflake unstructured types nested in a structured type (in an object) are converted.